### PR TITLE
#162342273 Creates a model for the Article.

### PR DESCRIPTION
**Pull request by gbols**
#### What does this PR do?
Creates the model for an article
#### Description of Task to be completed?
- define the relationships between the users model and other models
#### How should this be manually tested?
none
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
#[162342273](https://www.pivotaltracker.com/story/show/162342273)
#### Screenshots (if appropriate)
none

#### Questions:
none

### CODE REVIEW

File: migrations/20181202230331-create-profile.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
```
Comment (V-84) [inline migrations/20181202230331-create-profile.js]: nice work gbols. however, your this migration has no reference to the user model.

File: models/profile.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (V-84) [inline models/profile.js:1]: this profile model should be in a new branch

File: migrations/20181202230331-create-profile.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      allowNull: false,
+    },
+    createdAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    },
+    updatedAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    }
```
Comment (V-84) [inline migrations/20181202230331-create-profile.js:52]: you should reference user model

Comment (gbols) [reply migrations/20181202230331-create-profile.js:52]: thanks that has been done

**Review by efalayi — APPROVED** [review]

**Pull request merged**